### DISCUSSION TITLE
Is it possible to have monitor only by default but then opt in full auto update with labels?

### DISCUSSION THREAD

**Discussion by dutts** [containrrr/watchtower#1698] [General] [answered]
I like using watchtower to notify me when one of my containers needs updating, but I prefer to handle the update myself manually to make sure I can cover off any config changes or whatever that are required.

However, I have started hosting a Minecraft bedrock container on my home machine which seems to have quite regular updates (and won't let you-lay unless it is updated), for this reason I'd like to have watchtower update this container automatically for me. 

I see that you can enforce some watchtower behaviour through container labels but can't see how I could achieve what I'm after, is this possible?

Rich

**Comment by dutts** [comment]
Is this not possible then?

**Comment by piksel** [comment, marked as answer]
Actually, this feature landed in v1.6.0 in the form of https://containrrr.dev/watchtower/arguments/#with_label_taking_precedence_over_arguments

Where you can set watchtower to `monitor-only` mode and then put a label on the Minecraft server to opt back in to updates.

**Reply by dutts** [reply]
Ah that’s brilliant, I’ll try that this evening, thanks.
